File: app/src/main/java/com/jeanpigomez/codingtest/model/database/AppDatabase.java
package com.jeanpigomez.codingtest.model.database;

import android.arch.persistence.room.Database;
import android.arch.persistence.room.RoomDatabase;
import android.arch.persistence.room.TypeConverters;

import com.jeanpigomez.codingtest.model.Language;
import com.jeanpigomez.codingtest.model.LanguageDao;

@Database(entities = Language.class, version = 1)
@TypeConverters({Converters.class})
public abstract class AppDatabase extends RoomDatabase {

    public abstract LanguageDao languageDao();
}

File: app/src/main/java/com/jeanpigomez/codingtest/model/database/AppDatabase.kt
package com.jeanpigomez.codingtest.model.database

import android.arch.persistence.room.Database
import android.arch.persistence.room.RoomDatabase
import com.jeanpigomez.codingtest.model.Language
import com.jeanpigomez.codingtest.model.LanguageDao

@Database(entities = [Language::class], version = 1)
abstract class AppDatabase : RoomDatabase() {
    abstract fun languageDao(): LanguageDao
}
